catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -516,10 +515,7 @@ public PropertiesMetadata filesetPropertiesMetadata() throws UnsupportedOperatio
   public void close() throws IOException {}
 
   private Map<String, String> addManagedFlagToProperties(Map<String, String> properties) {
```

Comment:
Why don't you delete this method directly if it is not required?

Reply:
fixed